deploy/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+wget --show-progress -qO agent https://github.com/last9/k8stream/releases/${VERSION:-latest}/download/k8stream
+chmod a+x agent
```

Comment:
`agent` is an extremely ambiguous name. Please call it something more appropriate.